Muvr/MRSummaryExercise.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  MRSummaryExercise.swift
+//  Muvr
+//
+//  Created by Duc Hoang on 01/12/2015.
+//  Copyright © 2015 Muvr. All rights reserved.
+//
+
+import Foundation
+
+class MRSummaryExercise {
```

Comment:
The struct object is copied when I find one element with `filter` function and modify that object (in the function `aggregateClassifiedExercises()`).

https://github.com/muvr/muvr-ios/blob/feature/new_phone_UI/Muvr/Entities/MRManagedExerciseSession%2BHelper.swift#L66